style.css

```diff
@@ -0,0 +1,720 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: 'Poppins', sans-serif;
+}
+
+/* ------------------navbar section------------------- */
+.nav-container {
+  display: flex;
+  width: 1440px;
+  height: 72px;
+  background-color: #3e3c3c;
+  align-items: center;
+}
+
+.nav-container a {
+  text-decoration: none;
+  color: white;
+  font-family: 'Roboto', sans-serif;
+  padding-left: 20px;
+}
```

Comment:
As the previous reviewer requested, please make sure you are following the Figma template. Kindly check the differences below. (font-weight and font-size)

<img width="352" alt="Screenshot 2023-01-16 at 9 06 51 PM" src="https://user-images.githubusercontent.com/25887253/212742768-e6cb6354-5487-42ab-bcdf-441993c4e73c.png">
<img width="725" alt="Screenshot 2023-01-16 at 9 09 02 PM" src="https://user-images.githubusercontent.com/25887253/212742775-4a602573-464f-446b-852d-cd7c8d8776ac.png">